helpers/orderBy.js

```diff
@@ -77,4 +77,29 @@ function OrderBy (orderBy, options) {
 	};
 }
 
+/**
+ * Converts orderBy queries from string format like "field:asc"
+ * to format accepted by "json-sql" library: {field: 1}.
+ * Ascending sort method number equivalent is 1.
+ * Descending sort method number equivalent is -1.
+ * @param {string} sortQuery
+ * @param {Array} sortableFields
+ * @returns {Object}[={}] returns {} if incorrect format of sortQuery given or if field
+ */
+OrderBy.sortQueryToJsonSqlFormat = function (sortQuery, sortableFields) {
+	var sortQueryMatched = typeof sortQuery !== 'string' ? null : sortQuery.match(/^([a-zA-Z0-9]+):(asc|desc)$/);
+	if (!sortQueryMatched || sortableFields.indexOf(sortQueryMatched[1]) === -1) {
+		return {};
+	}
+	var sortField = sortQueryMatched[1];
+	var sortMethodsToNumbersMap = {
+		asc: 1,
+		desc: -1
+	};
+	var result = {};
+	var sortMethod = sortQueryMatched[2];
+	result[sortField] = sortMethodsToNumbersMap[sortMethod];
+	return result;
```

Comment:
I suggest that use of above variables are unnecessary and creating less readable code. Instead this line can work well for the purpose. 
```
return {[sortField]: sortMethodsToNumbersMap[sortQueryMatched[2]]};
```

Reply:
As you are aware we are not using ES6 features.

Reply:
I don't think its an ES6 feature.